operate/services/service.py

```diff
@@ -428,7 +429,7 @@ def _build_docker(
         force: bool = True,
         chain: t.Optional[str] = None,
     ) -> None:
-        """Build docker deployment."""
+        """Build docker deployment and kubernetes manifest."""
```

Comment:
this is `_build_docker` function.
we need to add a similar function, `_build_kubernetes`. And call it in the required places. You will find those `use_docker` parameters, similarly we need to add `use_kubernetes` parameter, False by default. And only build k8s from the QS flow.

Reply:
let's keep this function untouched for now, to avoid risk of breaking anything on Pearl.